src/components/CookieNotice/CookieNotice.tsx

```diff
@@ -0,0 +1,132 @@
+import "./CookieNotice.scss";
+import React from "react";
+import Backdrop from "@material-ui/core/Backdrop";
+import store from "store";
+import {useSelector} from "react-redux";
+import {ApplicationState} from "types/store";
+import {ActionFactory} from "store/action";
+import CookiePolicy from "./CookiePolicy";
+
+/**
+ * Converts string value to boolean value.
+ * @param input string that needs conversion to boolean value.
+ */
+const stringToBoolean = (input: string): boolean => {
```

Comment:
This function seems overly complicated to me, since you're storing the value `true` within your code and my guess is that noone would manually edit this value. I would rather suggest to replace that function with `cookieConsent.value === 'true'`